src/index.js

```diff
@@ -0,0 +1,223 @@
+    message: "What is the description of your project?",
+    name: "projectDescription",
+    validate(answers) {
+      if (!answers) {
+        return "Please enter a description.";
+      }
+      return true;
+    },
+  },
+  {
+    //confirm installation details
+    type: "confirm",
+    message: "Would you like to add project installation details?",
+    name: "confirmInstall",
+  },
+  {
+    //confirm installation details
+    // when: "confirmInstall",
+    type: "input",
+    message: "Enter installation details here:",
+    name: "projectInstall",
+    when(answers) {
+      return answers.confirmInstall;
+    },
+  },
+  {
+    //drop down of license options
+    type: "list",
+    message: "Please select the license for your project",
+    name: "projectLicense",
+    choices: [
+      "Apache",
+      "GNU General Public",
+      "MIT",
+      "BSD 2-Clause Simplified",
+      "BSD 3-Clause",
+      "Boost Software",
+      "Creative Commons Zero",
+      "Eclipse Public 2.0",
+      "None",
+    ],
+  },
+  {
+    //conditional
+    // when: "confirmContribution",
+    type: "confirm",
+    message: "Would you like to enter contribution guidelines for the project?",
+    name: "confirmContribution",
+  },
+  {
+    //conditional
+    // when: "confirmContribution",
+    type: "input",
+    message: "Please enter contribution guidelines for the project",
+    name: "projectContribution",
+    when(answers) {
+      return answers.confirmContribution;
+    },
+  },
+  {
+    //conditional
+    // when: "confirmUsage",
+    type: "confirm",
+    message: "Would you like to add usage information for the project?",
+    name: "confirmUsage",
+  },
+  {
+    //conditional
+    // when: "confirmUsage",
+    type: "input",
+    message: "Please enter usage information for the project",
+    name: "projectUsage",
+    when(answers) {
+      return answers.confirmUsage;
+    },
+  },
+  {
+    //conditional
+    // when: "confirmTests",
+    type: "confirm",
+    message:
+      "Would you like to add information on how a user could test the project?",
+    name: "confirmTests",
+  },
+  {
+    //conditional
+    // when: "confirmTests",
+    type: "input",
+    message: "Please enter information on how a user could test the project:",
+    name: "projectTests",
+    when(answers) {
+      return answers.confirmTests;
+    },
+  },
+  {
+    type: "input",
+    message: "What is your github username?",
+    name: "projectUsername",
+    validate(answers) {
+      if (!answers) {
+        return "Please enter a your github username.";
+      }
+      return true;
+    },
+  },
+  {
+    type: "input",
+    message: "What is your email address?",
+    name: "projectEmail",
+    validate(answers) {
+      if (!answers) {
+        return "Please enter your email address.";
+      }
+      return true;
+    },
+  },
+];
+
+const getBadge = (license) => {
+  if (license === "None") return "";
+
+  let badgeUrl;
+
+  switch (license) {
```

Comment:
What about all the other possible licenses? At the moment, if we select any licence other than MIT or Apache, the `badgeUrl` is undefined as we don't have any switch case statements for them.